integration/messaging/download-browsers.js

```diff
@@ -18,14 +18,9 @@
 const seleniumAssistant = require('selenium-assistant');
 
 console.log('Starting browser download - this may take some time.');
-Promise.all([
-  seleniumAssistant.downloadLocalBrowser('chrome', 'stable', 48),
-  seleniumAssistant.downloadLocalBrowser('chrome', 'beta', 48),
-  seleniumAssistant.downloadLocalBrowser('chrome', 'unstable', 48),
-  seleniumAssistant.downloadLocalBrowser('firefox', 'stable', 48),
-  seleniumAssistant.downloadLocalBrowser('firefox', 'beta', 48),
-  seleniumAssistant.downloadLocalBrowser('firefox', 'unstable', 48)
-])
+// TODO: enable firefox testing once figure out how to give notification permission with SE webdriver.
+// TODO: Run the integration test against multiple major chrome versions to ensure backward compatibility
+Promise.all([seleniumAssistant.downloadLocalBrowser('chrome', 'stable', 80)])
```

Comment:
Just thinking out loud, I think it would be more flexible to just use whatever chrome/firefox version that the machine has, so it will never get outdated. 

And for testing against multiple browsers/versions,  can we make saucelab drive the messaging IT and tell it what browser to use, @hsubox76 ?